src/main/java/common/tileentities/GTMTE_LapotronicSuperCapacitor.java

```diff
@@ -309,27 +299,87 @@ public boolean checkMachine(IGregTechTileEntity thisController, ItemStack guiSlo
 			}
 			mEnergyTunnelsTT.clear();
 			mDynamoTunnelsTT.clear();
-
 		}
 
+		//Check if enough (more than 50%) non-empty caps
+		int emptyCheck = capacitors[5]/(capacitors[0] + capacitors[1] + capacitors[2] + capacitors[3] + capacitors[4] + capacitors[6]);
```

Comment:
This is not correct. You need to cast them to floating point numbers.
```java
double emptyCheck = ((double)capacitors[5])/(double)(capacitors[0] + capacitors[1] + capacitors[2] + capacitors[3] + capacitors[4] + capacitors[6]);

Reply:
Fixed.